### PR TITLE
Update index.html

### PR DESCRIPTION
**Pull request by shaikhhamza11**

### CODE REVIEW

**Comment by Mohitkumar33** [comment]
Every thing is good. One thing that you should add is you can add about your thought process in the description about the approach you followed. Overall nice, the one thing I like is you organized your css very nicely and specially the root selector.
nice man.

**Comment by sks-21** [comment]
The code looks nice. One thing you could do is mention all the properties in alphabetical order inside any declaration box. That's optional but would make it more readable. Good Work!

**Comment by shaikhhamza11** [comment]
> Every thing is good. One thing that you should add is you can add about your thought process in the description about the approach you followed. Overall nice, the one thing I like is you organized your css very nicely and specially the root selector. nice man.

**Comment by shaikhhamza11** [comment]
> Every thing is good. One thing that you should add is you can add about your thought process in the description about the approach you followed. Overall nice, the one thing I like is you organized your css very nicely and specially the root selector. nice man.

Thank you so much. I will add that

**Comment by shaikhhamza11** [comment]
> The code looks nice. One thing you could do is mention all the properties in alphabetical order inside any declaration box. That's optional but would make it more readable. Good Work!

Thank you so much. Regarding mentioning the properties , you mean to add all the properties  used in css using comments??

File: css/encore.css [hunk 1]
```diff
@@ -6,24 +6,32 @@
...
+li {
+  list-style-type: none;
+}
+a {
+  text-decoration: none;
 }
 
 /* Css Variables */
 /* Defining all the main variables in root */
 :root {
```
Comment (nishasen) [inline css/encore.css:27]: Hey, everything looks great and neat. One thing that I think is you can define all the main variables in root at top.

Comment (shaikhhamza11) [reply css/encore.css:27]: Thanks . Done

**Pull request closed without merging**